.github/CODEOWNERS

```diff
@@ -2,8 +2,9 @@
 /src/snowflake/snowpark/modin/ @snowflakedb/snowpandas
 /tests/integ/modin/ @snowflakedb/snowpandas
 /tests/unit/modin/ @snowflakedb/snowpandas
+/docs/modin_api_coverage/ @snowflakedb/snowpandas
+/docs/source/modin/ @snowflakedb/snowpandas
 /.github/ @snowflakedb/snowpandas @snowflakedb/snowpark-python-api-reviewers
-/docs/ @snowflakedb/snowpandas @snowflakedb/snowpark-python-api-reviewers
```

Comment:
I feel this has a side-effect of removing `snowflakedb/snowpark-python-api-reviewers` as reviewer for files other than the `modin_api_coverage` and `modin` dir?

Reply:
@sfc-gh-aling should I add the line back? I can also add
`/docs/snowpark @snowflakedb/snowpark-python-api-reviewers` to the CODEOWNERS file

Reply:
CODEOWNERS rules get implemented in order, so it shouldn't since: `* @snowflakedb/snowpark-python-api-reviewers` is the first line (i.e. all files in this repo). So the default should be Snowpark Python reviewers for all other files in `docs/*` other than the two directories specifically tied to Snowpark pandas.